app/controllers/application_controller.rb

```diff
@@ -3,38 +3,32 @@ class ApplicationController < ActionController::Base
   before_filter :add_extra_sentry_metadata
   protect_from_forgery with: :exception
 
-  def self.permit_only_from_prisons
-    before_filter :reject_untrusted_ips!
-  end
-
   def self.permit_only_from_prisons_or_with_key
```

Comment:
This is literally what it does, but perhaps the important intention is slightly different: `permit_only_trusted_users` or similar (where trusted means either on a trusted IP or authenticated).